airbyte-webapp/src/components/ui/Callout/Callout.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from "classnames";
+
+import styles from "./Callout.module.scss";
+
+interface CalloutProps {
+  className?: string;
+  variant?: "yellow" | "red" | "blue";
```

Comment:
Not in use anywhere yet, but it is in some Figma designs so am adding it now to use shortly.

Reply:
Is there some semantic meaning we can attach instead of color names? Like `warning | error | default`? This could help us consistently use them for the right reason, and also makes changing the color name later easier, if we want to!

Reply:
Yep! Happy to switch that back.

Reply:
https://github.com/airbytehq/airbyte/pull/21273/commits/657236cad5c5918135215da8906b0260e76c3a67